### DISCUSSION TITLE
Lessons in Adminning

### DISCUSSION THREAD

**Discussion by Pixeltheaertist** [Pixeltheaertist/Trieste-Port-14-DEPRACATED#128] [Q&A]
Hello Adminlings, this guide is here to help teach some basics of adminning for Trieste Port. Mainly for Trialmins.

# BASICS

As an Admin, it's your job to make sure shitters aren't ruining the round or running amok destroying things. Lots of helpful tools at our disposal to help combat that.

**The Ahelp Relay**
The Ahelp Relay lets players send a message to online admins if they see someone breaking the rules or being a shitter. Very useful. People also tend to occasionally use it to request things or chat, you can generally ignore those if you want unless it's about an emerging issue. 

The relay also lets you see antagonists and message players if you need to chat with them. You can also kick and ban from the relay, as well as leave notes on a player's behavior. 

The relay also automatically sends ahelps from the server to our Ahelp channel, so if a developing situation is happening an admin can hop on to assist.

**Right Click Abilities**
When you right click an object or player, the normal interface pops up along with some new options. These include things such as:

- Debug
- Smites
- Antagonist stuff
- Viewing variables/components
- Tricks

In debug, you can view (and edit) reagents inside a person or object. You can also rejuvenate characters. Most buttons are self-explanatory.

In the Admin menu, you can respawn people by either sending them to the lobby or spawning them as a clone at your current location. Only spawn them as a clone if something really bad happened that led to their body being gone forever without antagonist involvement. Also make sure to tell them to respawn as a new character if sent to lobby, please.

If there is a situation where (for example) a suspected raider is killing someone and gibbing them in maints, use your right click panel to freeze them (stopping movements and attacking) and then Ahelp them to figure out what is happening. There's also a freeze + mute button if they start spamming the chat. Don't jump to conclusions for raiding or major situations, we can always check logs after. Better to ban someone later than ban the wrong bystander.

In **Tricks**, you can do some different things. Under a battery, you can either fill it, drain it, or give it infinite power. Useful if Trieste is out of power and no engis go to fix it. You can also give guns infinite ammo and remove their firing delay. You can also rename and redescribe items, among other things. 

One thing in Tricks is the ADMIN ARENA, which is the upwards pointing arrow. It teleports the player or object to your admin arena. Useful for admeme things for gathering players, testing, etc. 

# ADMIN PANEL

From the admin panel, you can see the ahelp relay, plus some other tools and random things like grids and maps for easier traversal around the game map.
In the Admin tab, you can do things like call the emergency shuttle or announce things. Announcing things is fun for Adminbus rounds to let players know what is/going to be happening.

# USEFUL COMMANDS

Some useful commands to know as an admin are 

- Fixgridatmos [Grid EntID] (should auto fill in the ID)
> Sets a grid's atmos to a roundstart mix. If there's irreparable flooding, extreme heat, or anything that is *genuinely* ruining the round and can't be fixed by players, run this command to reset the atmos system on your grid.
- Adminlogs
> Open the adminlogs panel, letting you see what players did what, said what, opened what, etc. Useful for figuring out who did a shitter activity if you only see the aftermath of it.
- Addgamerule [GAMERULE]
> Adds an event to the game. For example, the command addgamerule driftermigration spawns drifters at the edge of the platform. Respawndead allows players to respawn after dying.
- Delayroundstart/end
> Delays the start/end of a round
- Startround
> Manually starts a round
- Golobby
> Restarts the round immediately, sending all players to the lobby (NOT JUST YOU). Don't accidentally run this, I have.
- Readmin/Deadmin
> Gives/Revokes your admin privileges. This includes ahelp, commands, etc. **If you are playing in a round, you MUST deadmin yourself.**
- [optional: sudo] cvar [CVAR]
> Changes a CVAR for the server. For example, changing the diving bell/ocean map, changing atmos, changing certain game systems or rules. Generally, doesn't need to be touched for the most part.
- Motd
> Sets the message of the day. Useful for telling players joining the server certain things. Example: **motd DO NOT PLAY AS A JELLID. YOU WILL DIE.** Stuff like that.

# SMITES

Smites are **punishments**. For the most part, if you're smiting someone, they are doing something terribly wrong and are probably getting a note or getting kicked/banned. Feel free to pick your favorite smite, they all tend to do the same sort of thing: causing the shitter to suffer.

There's a lot of smites. Bonking them on a table, or every table in the game, cutting their hands off, making them into a cat maid, exploding them, throwing them into the abyss, making them bread. Good times. 

# ADMIN RULES

As an Admin, your job is to observe the round and make sure nothing is breaking. You can interfere occasionally if it would help the round, and answer prayers if you want to. But for the most part, observe and fix issues that inevitably arise within the round.

Some general guidelines are things like:
- Look at both sides of a story + logs before making a decision on kicking, banning, notes, etc.
- You don't HAVE to answer ahelps if they are pointless.
- Admemes are okay in moderation, if you're going to do an Adminbus round alert other admins beforehand so a plan for the round can be made.
- You are expected to know the server rules as an Admin, so be sure to review them. They are pretty similar to Wizden with some minor changed. I will be pinging if rules are ever updated, so don't worry about having to check all the time.

Don't be a dick, as you know. As a figurehead of the server, people will base their opinions on the server based on past admin interactions. Be stern, but don't be mean.

# EXTRA TIPS

If a Rotund Stepfather has no pilot, you can assign them one. Do this by viewing the variables of your selected character and clicking server components. Then click "add" and add the component named "expedpilot". This will allow the assigned Pilot to control the Rotund Stepfather.

Currently, the only fix for the diving bell breaking (not docking to sweetwater) is to use the golobby command to restart the round. People don't like it, but the issue isn't fixed yet.

Roundstart, the ocean floor and platform don't have weather. Use the weather command (followed by the map number, viewable from the f3 menu) and then do the name for the selected weather (normal one is rain).

Also roundstart, the ocean surface and platform aren't connected. Use a hand teleporter at the start of the round to connect them with portals until a better system is implemented.

Syndies are also currently not working, as they get a ringer code but can't punch it in. Give Syndies a 20 TC radio if they are syndicate and not NT.

Please put questions and comments below if you are an admin

**Comment by MrCheese2248** [comment]
wait so at the start of every round you have to set the weather and make sure the bell works

**Reply by Pixeltheaertist** [reply]
For now yes until I fix those, ideally they’ll do it themselves in the future
